net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```
@@ -144,23 +181,40 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		--without-seccomp
```

We could add a comment there:
```suggestion
        # Remove by upstream in 26.0 with https://github.com/bitcoin/bitcoin/commit/32e2ffc39374f61bb2435da507f285459985df9e
		--without-seccomp
```